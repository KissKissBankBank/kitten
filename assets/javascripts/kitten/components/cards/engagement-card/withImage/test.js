import React from 'react'
import renderer from 'react-test-renderer'
import { EngagementCardWithImage } from './index'

describe('<EngagementCardWithImage />', () => {
  let component

  describe('by default', () => {
    beforeEach(() => {
      component = renderer
        .create(
          <EngagementCardWithImage
            backgroundImage="https://source.unsplash.com/WLUHO9A_xik/200x120"
            href="#"
            title="Hello word"
          />,
        )
        .toJSON()
    })

    it('matches with snapshot', () => {
      expect(component).toMatchSnapshot()
    })
  })
})
