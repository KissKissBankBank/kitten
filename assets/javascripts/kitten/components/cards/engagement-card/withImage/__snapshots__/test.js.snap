// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<EngagementCardWithImage /> by default matches with snapshot 1`] = `
<a
  className="withImage__StyledContainer-sc-1jg09sq-0 iuidGJ"
  href="#"
  title="Hello word"
>
  <div
    className="withImage__StyledCard-sc-1jg09sq-1 lgohnN"
  >
    <img
      className="withImage__StyledImage-sc-1jg09sq-2 fWPyYx"
      src="https://source.unsplash.com/WLUHO9A_xik/200x120"
    />
  </div>
  <p
    className="k-u-color-font1 k-u-decoration-none k-u-size-micro k-u-weight-regular withImage__StyledText-sc-1jg09sq-3 ihfHEi"
  />
</a>
`;
